core/src/main/java/io/questdb/griffin/engine/groupby/AbstractNoRecordSampleByCursor.java

```diff
@@ -108,6 +114,9 @@ public void of(RecordCursor baseCursor, SqlExecutionContext executionContext) th
         rowId = 0;
         isNotKeyedLoopInitialized = false;
         areTimestampsInitialized = false;
+        endFill = false;
+        sampleFromFunc.init(null, executionContext);
```

Comment:
There is no point in not providing the symbol source here and in the below init call:
```suggestion
        sampleFromFunc.init(baseCursor, executionContext);
```